### PR TITLE
Deprecated 'influx transpile' command

### LINKED ISSUE #2266: Deprecate 'influx transpile' command
The `influx transpile` command will soon be removed from the `influx` CLI. Because it still exists in previous patch version of 2.0, we shouldn't just remove the `influx transpile` doc. We need to do the following:

- Remove the `transpile` command from the `influx` CLI landing pages
- Add deprecation/removal warning messages to the `influx transpile` pages that identify the specific version the command will be / was removed.
- Remove the `menu` frontmatter from each of the `influx transpile` pages to hide it from the the menu, but still publish the page.

Essentially there won't be any way to navigate to the page, but external links will still work and people may still be able to find it if they're looking for it, but we make it clear that it's no longer available after InfluxDB 2.0.x.

##### Relevant URLs
- https://github.com/influxdata/influxdb/issues/20905
- https://docs.influxdata.com/influxdb/cloud/reference/cli/influx/
- https://docs.influxdata.com/influxdb/v2.0/reference/cli/influx/
- https://docs.influxdata.com/influxdb/cloud/reference/cli/influx/transpile/
- https://docs.influxdata.com/influxdb/v2.0/reference/cli/influx/transpile/

### PR DESCRIPTION
**Pull request by sanderson**
Closes #2266

- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/v2.0/reference/cli/influx/transpile/_index.md [hunk 1]
```diff
@@ -2,17 +2,24 @@
...
 weight: 101
-influxdb/v2.0/tags: [influxql, flux]
 related:
   - /influxdb/v2.0/reference/cli/influx/#provide-required-authentication-credentials, influx CLI—Provide required authentication credentials
   - /influxdb/v2.0/reference/cli/influx/#flag-patterns-and-conventions, influx CLI—Flag patterns and conventions
 ---
 
+{{% warn %}}
+### Deprecated in InfluxDB OSS 2.0.5
+The `influx transpile` command was deprecated in **InfluxDB 2.0.5** in favor of
```
Comment (danxmoran) [inline content/influxdb/v2.0/reference/cli/influx/transpile/_index.md:13]:
```suggestion
The `influx transpile` command was removed in **InfluxDB 2.0.5** in favor of
```
"deprecated" makes me think the command is still there.

File: content/influxdb/v2.0/reference/cli/influx/transpile/_index.md [hunk 2]
```diff
@@ -2,17 +2,24 @@
...
+[using InfluxQL to query in InfluxDB 2.0](/influxdb/v2.0/query-data/influxql/).
+For information about manually converting InfluxQL queries to Flux, see:
+
+- [Get started with Flux](/influxdb/v2.0/query-data/get-started/)
+- [Query data with Flux](/influxdb/v2.0/query-data/flux/)
+- [Migrate continuous queries to Flux tasks](/influxdb/%762.0/upgrade/v1-to-v2/migrate-cqs/)
+  <!-- Used the hex code in the link above to prevent v2.0 -> cloud replacement -->
```
Comment (kelseiv) [inline content/influxdb/v2.0/reference/cli/influx/transpile/_index.md:20]: folks can't do in cloud?

File: content/influxdb/v2.0/reference/cli/influx/transpile/_index.md [hunk 3]
```diff
@@ -2,17 +2,24 @@
...
-influxdb/v2.0/tags: [influxql, flux]
 related:
   - /influxdb/v2.0/reference/cli/influx/#provide-required-authentication-credentials, influx CLI—Provide required authentication credentials
   - /influxdb/v2.0/reference/cli/influx/#flag-patterns-and-conventions, influx CLI—Flag patterns and conventions
 ---
 
+{{% warn %}}
+### Deprecated in InfluxDB OSS 2.0.5
+The `influx transpile` command was deprecated in **InfluxDB 2.0.5** in favor of
+[using InfluxQL to query in InfluxDB 2.0](/influxdb/v2.0/query-data/influxql/).
```
Comment (kelseiv) [inline content/influxdb/v2.0/reference/cli/influx/transpile/_index.md:14]:
```suggestion
The `influx transpile` command was removed in **InfluxDB 2.0.5.** 
[Use InfluxQL to query in InfluxDB 2.0](/influxdb/v2.0/query-data/influxql/).
```

**Review by kelseiv — APPROVED** [review]

File: content/influxdb/v2.0/reference/cli/influx/transpile/_index.md (hunk 2 above)
Comment (sanderson) [reply content/influxdb/v2.0/reference/cli/influx/transpile/_index.md:20]: The "Migrate CQs to tasks doc" doesn't exist in cloud (yet), so this ensures the URL doesn't get replaced by the duplicate-oss shortcode when it copies over the content and replaces all the `v2.0` links.

**Pull request merged**